backend/inspirehep/submissions/views.py

```diff
@@ -259,12 +263,31 @@ def create_ticket(self, record, rt_template):
         )
 
 
+class ExperimentSubmissionsResource(BaseSubmissionsResource):
+    def load_data_from_request(self):
+        return experiment_loader_v1()
+
+    def post(self):
+        """Adds new experiment record"""
+
+        request_data = self.load_data_from_request()
+        data = {
+            "_collections": ["Experiments"],
+            "$schema": "https://inspirehep.net/schemas/records/experiments.json",
+            "legacy_name": request_data["legacy_name"],
+            "project_type": request_data["project_type"],
+        }
+        record = ExperimentsRecord(data=data).create(data)
+        db.session.commit()
+        experiment_serializer_v1.dump(record)
```

Comment:
why you need it?

Reply:
Don't I need to serialize the record or is it okay without serialization? On the other hand, I'm returning just the control number and status.